orbit/pkg/update/update.go

```diff
@@ -553,3 +556,218 @@ func (u *Updater) initializeDirectories() error {
 
 	return nil
 }
+
+type FileMigration struct {
+	OldPath string
+	NewPath string
+}
+
+// MigrateRoot mirates a previous installation to the new-style paths (linux only). If it returns true, then a migration
```

Comment:
Ah I thought we agreed no migration for existing installations was needed (for reduced risk).